src/otx/recipe/anomaly_classification/uflow.yaml

```diff
@@ -0,0 +1,44 @@
+model:
```

Comment:
Since we intend to use only "Anomaly" task in the future, maybe it is not needed to support other tasks for the new models.